apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/account-permissions/components/account-signer.tsx

```diff
@@ -46,16 +47,12 @@ export const AccountSigner: React.FC<AccountSignerProps> = ({
             </Heading>
             <div className="flex flex-row gap-2">
               {isAdmin ? (
-                <Badge borderRadius="lg" p={1.5}>
-                  Admin Key
-                </Badge>
+                <Badge className="rounded-lg p-1.5">Admin Key</Badge>
               ) : (
-                <Badge borderRadius="lg" p={1.5}>
-                  Scoped key
-                </Badge>
+                <Badge className="rounded-lg p-1.5">Scoped key</Badge>
```

Comment:
Remove the padding and rounded classes here - just use the default styling

Reply:
thanks! I haven't tested this PR yet (the local dashboard thingy is not running - will check later today)